apps/website-example/server.ts

```diff
@@ -117,7 +117,7 @@ export async function runServerInstance() {
 
 if (process.env.NODE_ENV !== 'development') {
   if (cluster.isMaster) {
-    const numWorkers = process.env.NUM_WORKERS || os.cpus().length
+    const numWorkers = +(process.env.NUM_WORKERS || os.cpus().length)
```

Comment:
Nx automatically updated typescript to 5.0 but I downgraded it until 5.1 is out (as it fixes an issue we are facing that started to happen in 4.7). This was something typescript complained about as it might not be a number